src/User.php

```diff
@@ -7392,4 +7392,238 @@ private function cleanInput(array $input): array
 
         return $input;
     }
+
+    /**
+     * Count the number of computers used by a user
+     *
+     * @param array $sql_filters SQL filters to apply
+     *
+     * @return integer
+     */
+    public function countComputersForUser(array $sql_filters = [])
+    {
+        /** @var \DBmysql $DB */
+        global $DB;
+        $user_name = $this->getField('name');
+
+        $query = [
+            'COUNT'      => 'cpt',
+            'FROM'       => 'glpi_logs',
+            'WHERE'      => [
+                'itemtype'         => 'Computer',
+                'OR' => [
+                    'glpi_logs.old_value' => ['LIKE', "%$user_name%"],
+                    'glpi_logs.new_value' => ['LIKE', "%$user_name%"]
```

Comment:
I’m not a fan of using `LIKE`, especially when searching for names that may change, in a table that can quickly grow with many rows. I believe the search would be more optimized if we were to search by IDs. For this, we would likely need to add two columns, 'old_id' and 'new_id', to perform direct searches without using LIKE.

What do you think? @cedric-anne @trasher @AdrienClairembault

_If this makes sense, it's probably an improvement to make in a dedicated PR._

Reply:
Clearly, using a `LIKE` condition on a `glpi_logs` column is not a good idea. This table can contain million rows.

Reply:
Indeed this will be a big performance issue